service/controller.go

```diff
@@ -2132,9 +2139,75 @@ func (s *service) GetCapacity(
 			"Unable to get capacity: %s", err.Error())
 	}
 
+	systemID := ""
+	for key, value := range params {
```

Comment:
@KshitijaKakde We have already retrieved the systemID from the request body so I believe we can remove this duplicate code here, what do you think?
![image](https://github.com/dell/csi-powerflex/assets/109620911/d99ef2e2-3c88-40fd-b3b8-06281055c632)